website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -203,6 +204,15 @@ You should also consider the following questions when creating preconditions and
 - Which approach is more convenient? If a particular resource has many dependencies that all make an assumption about that resource, it can be pragmatic to declare that once as a post-condition of the resource, rather than declaring it many times as preconditions on each of the dependencies.
 - Is it helpful to declare the same or similar conditions as both preconditions and postconditions? This can be useful if the postcondition is in a different module than the precondition because it lets the modules verify one another as they evolve independently.
 
+## Check Assertions
```

Comment:
So in this case what I want to refer to is the `assert` statements within the check blocks. This is because the assert block within a check block matches the syntax for the other custom conditions (pre/postconditions, input validations) discussed in this page.

Pending the other rename (Checks to Check blocks), we could rename this to Check block assertions instead? I would like to highlight that we are specifically talking about the assert sub block somehow.